charts/zookeeper-operator/values.yaml

```diff
@@ -23,3 +23,13 @@ crd:
 ## Specifies which namespace the Operator should watch over.
 ## An empty string means all namespaces.
 watchNamespace: ""
+
+#
+# Operator pod resources
+resources:
+  limits:
+    cpu: 2
+    memory: 256Mi
+  requests:
+    cpu: 1
+    memory: 128Mi
```

Comment:
It would be better to comment these values and provide an empty list as default. 
```
resources: {}
  #commented values for limits and requests
```
This way the defaults would be used generally, but can be uncommented if the user wants to set a specific value